### ISSUE TITLE
does kinect700 include kinect600 and kinect 400

### ISSUE THREAD

**Issue by MarStarck** [cvdfoundation/kinetics-dataset#48]

**Comment by joaoluiscarreira** [comment]
Yes, check the home page -- "Note that it may not be safe to train / test across different versions of the dataset, for example the k400 validation set is largely part of the k700 training set so results will not be meaningful."
